### PR TITLE
plan: implement ExplainInfo() interface for simple operators

### PR DESCRIPTION
**Pull request by zz-jason**
to https://github.com/pingcap/tidb/issues/3805

### CODE REVIEW

**Comment by zz-jason** [comment]
@hanfei1991 @lamxTyler @winoros PTAL

File: plan/physical_plans.go [hunk 1]
```diff
@@ -530,6 +530,18 @@ const (
 	CompleteAgg
 )
 
+func (at AggregationType) String() string {
```
Comment (lamxTyler) [inline plan/physical_plans.go]: Add comment for the exported function.
Also, it seems that this function is not used?

File: executor/analyze_test.go [hunk 2]
```diff
@@ -52,7 +52,7 @@ func (s *testSuite) TestAnalyzeTable(c *C) {
 	tk.MustExec("analyze table t1 index ind_a")
 	result = tk.MustQuery("explain select * from t1 where t1.a = 1")
 	rowStr = fmt.Sprintf("%s", result.Rows())
-	c.Check(strings.Split(rowStr, "{")[0], Equals, "[[TableScan_4 Selection_5 cop ] [Selection_5  cop ] [TableReader_6  root ]]")
+	c.Check(strings.Split(rowStr, "{")[0], Equals, "[[TableScan_4 Selection_5 cop ] [Selection_5  cop eq(col(test.t1.a), const(1))] [TableReader_6  root ]]")
```
Comment (lamxTyler) [inline executor/analyze_test.go]: The `Split` is unnecessary now, the same with line 42, 46.

File: plan/physical_plans.go (hunk 1 above)
Comment (zz-jason) [inline plan/physical_plans.go]: it is used by `PhysicalAggregation.ExplainInfo()`

File: executor/analyze_test.go (hunk 2 above)
Comment (zz-jason) [inline executor/analyze_test.go]: done

File: executor/analyze_test.go [hunk 3]
```diff
@@ -52,7 +52,7 @@ func (s *testSuite) TestAnalyzeTable(c *C) {
 	tk.MustExec("analyze table t1 index ind_a")
 	result = tk.MustQuery("explain select * from t1 where t1.a = 1")
 	rowStr = fmt.Sprintf("%s", result.Rows())
-	c.Check(strings.Split(rowStr, "{")[0], Equals, "[[TableScan_4 Selection_5 cop ] [Selection_5  cop ] [TableReader_6  root ]]")
+	c.Check(rowStr, Equals, "[[TableScan_4 Selection_5 cop ] [Selection_5  cop eq(col(test.t1.a), const(1))] [TableReader_6  root ]]")
```
Comment (hanfei1991) [inline executor/analyze_test.go]: I think col(blabla..), constant(blabla...) is redundant...

Comment (zz-jason) [inline executor/analyze_test.go]: removed

**Comment by zz-jason** [comment]
@hanfei1991 @lamxTyler PTAL again

**Pull request merged**